tools/socket_passing.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
```

Comment:
What about `/usr/bin/env python2.7` then? You can't assume Python is in `/usr/bin`.